src/locales/es.json

```diff
@@ -6,8 +6,13 @@
     "next_button": "Verificar identidad"
   },
   "document_selector": {
-    "title": "Verifique su identidad",
-    "hint": "Seleccione el tipo de documento que desea subir"
+    "identity": {
+      "title": "Verifique su identidad",
+      "hint": "Seleccione el tipo de documento que desea subir",
+      "passport_hint": "Página del pasaporte con su foto",
+      "driving_licence_hint": "Frente y reverso",
+      "national_identity_card_hint": "Frente y reverso"
+    }
```

Comment:
I can see a few strings missing, are they still to be translated?

Reply:
Translation is not needed for PoA documents for now, so I only added translations for those strings that are needed for ID documents